src/UI/Internal/FiltersHeaders.elm

```diff
@@ -87,7 +106,7 @@ headerApplied : RenderConfig -> msg -> msg -> String -> String -> Element msg
 headerApplied renderConfig openMsg clearMsg clearHint label =
     -- Button.primary
     Element.row (Element.onIndividualClick openMsg :: headerAttrs True)
-        [ Element.text label
+        [ Element.text (label ++ " (1)")
```

Comment:
I didn't get this, what is this?

Reply:
Me neither, I just saw it yesterday on Figma and didn't ask Sufyian yet.
It can be seen in "Col Header - FilterApplied, CTA Hovered"  on Figma.